react/Condition.ts

```diff
@@ -48,4 +57,59 @@ const Condition: FC<Props> = ({ children, conditions, match }) => {
   return (children as any) ?? null
 }
 
+Condition.schema = {
```

Comment:
Can we move this to `interfaces.json`? so we avoid loading this JSON in the storefront.

I think you should only have here a little schema with the title property so this blocks may appear in the site-editor